napari/layers/image_surface_mixin.py

```diff
@@ -0,0 +1,148 @@
+import warnings
+
+import numpy as np
+from vispy.color import Colormap
+
+from ..utils.colormaps import make_colorbar
+from ..utils.event import Event
+from ..utils.status_messages import format_float
+from ..utils.validators import validate_n_seq
+from .layer_utils import increment_unnamed_colormap
+
+
+validate_2_tuple = validate_n_seq(2)
+
+
+class ImageSurfaceMixin:
+    """A mixin that adds gamma, colormap, and contrast limits logic to Layers.
+
+    When usinging, this should come before the Layer in the inheritance, e.g.:
```

Comment:
```suggestion
    When used, this should come before the Layer in the inheritance, e.g.:
```

Reply:
oh, you don't know the word "usinging"?  must be an australian/american english thing... ;)